### ISSUE TITLE
Add plugin name to installation instructions in README.md (Plugin name is not mentioned in the installation instructions)

### ISSUE THREAD

**Issue by storrer** [timhor/obsidian-editor-shortcuts#74]
TLDR; I suggest adding the name of the plugin "Code Editor Shortcuts" to the installation section of the readme.

**Steps to reproduce:**

1. Open the Obsidian app.
2. Click the **Settings** button.
3. Click the **Community plugins** tab.
4. Search for the plugin "obsidian editor shortcuts".

**Expected behavior:**

The plugin should be visible in the list of plugins.

**Actual behavior:**

The plugin is not visible in the list of plugins.

**Proposed solution:**

Add the name of the plugin to the installation instructions section of the 
repository's README.md file. This will make it easier for users to find the plugin in the Obsidian plugin library.

**Thank you for your time and consideration.**

### More information explanation of why anyone should care
I found this plug in by first searching Google for a multi-cursor editor and then reading a thread on the Obsidian forum where this plugin was mentioned. The link looks like this: 

![image](https://github.com/timhor/obsidian-editor-shortcuts/assets/13088830/12b81fa0-890e-402e-a7e8-bb197ade5e26)

The name of the repository for the plugin is `obsidian-editor-shortcuts` but I can't find this plugin when I search the Obsidian library with the that name. Belatedly I found out that the plugin's name is written at the top of the README.md for the repository. However, I scrolled past the heading of the Readme to find the installation instructions, as one does, and they currently read:

> ## Installing the plugin
>Refer to the official installation instructions for third-party plugins [here](https://help.obsidian.md/Extending+Obsidian/Community+plugins).

It would be good to add the name of the plug in here in the installation instructions for others who have the same problem I did.

**Comment by timhor** [comment]
Thanks for the suggestion 🙂 this has been included in the latest  README
